docs/modules/operation/pages/deep-dive/admin/webui/dashboard.adoc

```diff
@@ -1,137 +1,154 @@
+For more information on configuring the surveillance view component, see <<deep-dive/admin/webui/surveillance-view.adoc#ga-surveillance-view, Surveillance View>>.
 
-.The Surveillance View forms the basis for the dashboard page.
+.A surveillance view component in its default state (all entries selected)
 image::dashboard/02_dashboard-surveillance-view.png[]
 
 [[ga-dashboard-alarms]]
 === Alarms
 
-The Alarms component gives an overview about all unacknowledged alarms with a severity higher than Normal(1).
-Acknowledged alarms will be removed from the responsibility of the operator.
-The following information are shown in:
+The alarms component displays an overview of all unacknowledged alarms whose severity is higher than Normal (1).
+It sorts alarms from most to least recent, and lets you scroll through the last 100 alarms.
 
-.Information displayed in the alarms component
-image::dashboard/03_dashboard-alarms.png[]
+Acknowledged alarms are removed from the operator's responsibility and, therefore, the alarms component.
+
+The following information is shown in the component:
 
-1. Node: Node label of the node associated with the alarm.
-2. Severity: Severity of the alarm.
-3. UEI: Shows the UEI of the alarm.
-4. Count: Number of alarms deduplicated by the reduction key of the alarm.
-5. Last Time: Time of the most recent alarm.
-6. Log Msg: The log message from the event definition.
+.An alarms component displaying three alarms--two of Minor severity, and one of Warning severity
+image::dashboard/03_dashboard-alarms.png[]
 
-The Alarms component shows the most recent alarms and lets the user scroll through the last 100 alarms.
+1. *Node:* the label of the node associated with the alarm.
+2. *Severity:* severity of the alarm.
+3. *UEI:* the alarm's UEI.
+4. *Count:* the number of alarms deduplicated by the alarm's reduction key.
+5. *Last Time:* the time at which the most recent alarm was generated.
+6. *Log Msg:* the log message from the alarm's event definition.
 
 [[ga-dashboard-notifications]]
 === Notifications
 
-To inform people on a duty schedule notifications are used and force action to fix or reconfigure systems immediately.
-In {page-component-title} it is possible to acknowledge notifications to see who is working on a specific issue.
-The dashboard should show outstanding notifications in the NOC to provide an overview and give the possibility for intervention.
+The notifications component shows outstanding and unacknowledged notifications.
+It sorts notifications from most to least recent, and lets you scroll through the last 100 notifications.
 
-.Information displayed in the Notifications component
-image::dashboard/04_dashboard-notifications.png[]
+Notifications are used to inform people on a duty schedule of issues in the monitored network, and to force action to fix or reconfigure systems immediately.
+In {page-component-title}, it is possible to acknowledge notifications to designate someone as working on a specific issue.
```

Comment:
This isn't quite clear. I think it means that you can designate someone as working on a specific issue as part of acknowledging a notification.

Reply:
I believe so, but I'd like to confirm with @mmahacek or @uquevedo.

Reply:
Acknowledging a notification removes them from the everyone's view in the web UI.  If you have multiple operators logged in, one user ack'ing a notice will hide it from others so they don't see the notice active as something to address.

Reply:
```suggestion
In {page-component-title}, you can acknowledge notifications to indicate that they are being worked on.
When a notification is acknowledged, it is removed from the component.
```